build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml

```diff
@@ -25,7 +25,8 @@ steps:
       - "buildkite-agent artifact download --step build build-artifacts.tgz ./"
       - "tar xzf build-artifacts.tgz"
       - "find build-android/ -name '*.cmake' -exec sed -i \"s!\\$IREE_DOCKER_WORKDIR/!\\$PWD/!g\" {} \\;"
-      - "ctest --test-dir build-android/ --timeout 900 --output-on-failure"
+      - "cd build-android/"
+      - "ctest --timeout 900 --output-on-failure"
```

Comment:
Argh, I even triggered this pipeline on presubmit before merging: https://buildkite.com/iree/iree-test-android/builds/6348... but I didn't double check that the logs showed tests being run >_>

For this partial revert I'll do the same: https://buildkite.com/iree/iree-test-android/builds/6406 , and will actually check the logs this time...

Reply:
Looks like we can also set 
> `--no-tests=<[error|ignore]>`
>   Regard no tests found either as error or ignore it.
> 
>   If no tests were found, the default behavior of CTest is to always log an error message but to return an error code in script mode only. This option unifies the behavior of CTest by either returning an error code if no tests were found or by ignoring it.

Reply:
Oh CMake... I found that flag at some point. The `--do-the-right-thing` flag. I've honestly been tempted for a while to write a ctest wrapper script that explicitly parses all the flags so you don't just get silent weird behavior when you pass it an unknown flag

Reply:
I'll send a separate PR to add the `--do-the-right-thing` flag